### PR TITLE
ASBS documentation review

### PR DESCRIPTION
**Pull request by mauroservienti**

### CODE REVIEW

File: samples/azure-service-bus-netstandard/send-reply/ASBS_1/Endpoint1/Program.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ static async Task Main()
         var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus_ConnectionString");
         if (string.IsNullOrWhiteSpace(connectionString))
         {
-            throw new Exception("Could not read the 'AzureServiceBus.ConnectionString' environment variable. Check the sample prerequisites.");
+            throw new Exception("Could not read the 'AzureServiceBus_ConnectionString' environment variable. Check the sample prerequisites.");
```
Comment (seanfarmar) [inline samples/azure-service-bus-netstandard/send-reply/ASBS_1/Endpoint1/Program.cs]: Should this be fixed in the sample code as well?

File: samples/azure-service-bus-netstandard/send-reply/ASBS_1/Endpoint2/Program.cs [hunk 2]
```diff
@@ -17,7 +17,7 @@ static async Task Main()
         var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus_ConnectionString");
         if (string.IsNullOrWhiteSpace(connectionString))
         {
-            throw new Exception("Could not read the 'AzureServiceBus.ConnectionString' environment variable. Check the sample prerequisites.");
+            throw new Exception("Could not read the 'AzureServiceBus_ConnectionString' environment variable. Check the sample prerequisites.");
```
Comment (seanfarmar) [inline samples/azure-service-bus-netstandard/send-reply/ASBS_1/Endpoint2/Program.cs]: Should this be fixed in the sample code as well?

File: samples/azure-service-bus-netstandard/native-integration/sample.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ snippet: NecessaryHeaders
 
 NOTE: The `NServiceBus.EnclosedMessageTypes` property must contain the the fully-qualified name of the type expected by the NServiceBus endpoint.
 
-The message itself is defined as an `IMessage` in the `Shared` project.
+The message itself is defined as message using [conventions](/nservicebus/messaging/conventions.md) in the `Receiver` project.
```
Comment (seanfarmar) [inline samples/azure-service-bus-netstandard/native-integration/sample.md]: Shall we add a snippet here with 
`endpointConfiguration.Conventions().DefiningMessagesAs(type => type.Name == "NativeMessage");`?

Comment (mauroservienti) [reply samples/azure-service-bus-netstandard/native-integration/sample.md]: good point, probably yes we should

File: samples/azure-service-bus-netstandard/send-reply/ASBS_1/Endpoint1/Program.cs (hunk 1 above)
Comment (mauroservienti) [reply samples/azure-service-bus-netstandard/send-reply/ASBS_1/Endpoint1/Program.cs]: the snippet already comes from the sample, so we're good.

File: samples/azure-service-bus-netstandard/send-reply/ASBS_1/Endpoint2/Program.cs (hunk 2 above)
Comment (mauroservienti) [reply samples/azure-service-bus-netstandard/send-reply/ASBS_1/Endpoint2/Program.cs]: the snippet already comes from the sample, so we're good.

File: samples/azure-service-bus-netstandard/native-integration/sample.md (hunk 3 above)
Comment (SeanFeldman) [reply samples/azure-service-bus-netstandard/native-integration/sample.md]: I'm not convinced we should. The focus of sample is about ASBS, not conventions. If we want to _link_ to conventions, we could do so w/o adding extra code snippets and stealing the focus from the main objective. Which is ASBS transport.

For example:
_Note: the sample is using Message Conventions. More about this feature [here](https://docs.particular.net/nservicebus/messaging/conventions)_

**Comment by mauroservienti** [comment]
Link to conventions is already in place. Removing the WIP

**Pull request merged**